### ISSUE TITLE
Script for Windows not working 

### ISSUE THREAD

**Issue by AbhimanyuAryan** [petvana/AutoSysimages.jl#3]
trying to follow this: https://petvana.github.io/AutoSysimages.jl/stable/install/#Script-for-Windows and getting the below error

```
C:\Users\aryan\Documents\AutoSysimages.jl\scripts>cd windows

C:\Users\aryan\Documents\AutoSysimages.jl\scripts\windows>./asysimg.bat
'.' is not recognized as an internal or external command,
operable program or batch file.

C:\Users\aryan\Documents\AutoSysimages.jl\scripts\windows>asysimg.bat
'"C:\Users\aryan\Documents\AutoSysimages.jl\scripts\windows\\julia.exe"' is not recognized as an internal or external command,
operable program or batch file.
```

**Comment by petvana** [comment]
The last one seems to work, but is unable to find Julia in your `PATH`. @AbhimanyuAryan Are you able to run Julia from the same terminal? When you install Julia using installer, there is an option to add it to the `PATH`.

Also, it should be possible to use both `asysimg.bat` or only `asysimg` for short. 

(I'm not big fan of Windows, so this may not be tested well, sorry for that.)

**Comment by AbhimanyuAryan** [comment]
@petvana I am using juliaup from Microsoft Store. Do you think that's an issue?

**Comment by petvana** [comment]
@AbhimanyuAryan
> @petvana I am using juliaup from Microsoft Store. Do you think that's an issue?

That is weird, it should be in the `PATH` according the docs in https://github.com/JuliaLang/juliaup
> Once you have installed Juliaup, julia is on the PATH, and on Windows there is a start menu shortcut and it will show up as a profile in Windows Terminal. Any of those will start Julia. The VS Code extension will also automatically find this Julia installation.

**Comment by AbhimanyuAryan** [comment]
@petvana looks like it can't access ENV variables

<img width="836" alt="dfdfd" src="https://user-images.githubusercontent.com/8083613/190858098-08c5333e-1a6e-493f-b91f-81e89809c3e3.png">

**Comment by petvana** [comment]
Xref: #14

**Comment by petvana** [comment]
@AbhimanyuAryan Seems this can be fixed by #17 that sets an absolute path to the Julia binary into `asysimg` script when calling `AutoSysimages.install()`.

**Comment by AbhimanyuAryan** [comment]
@petvana are you planning to merge this and tag new release?

**Comment by AbhimanyuAryan** [comment]
also why not use `Sys.BINDIR` for all of these? Checking on Ubuntu WSL with `juliaup`

https://github.com/petvana/AutoSysimages.jl/blob/c8bfb5dccced031d0e6701dbebb3534864c6129b/src/AutoSysimages.jl#L377-L387

I think a lot of people use `juliaup` instead of installing directly using deb, exe or dmg

**Comment by ufechner7** [comment]
> I think a lot of people use `juliaup` instead of installing directly using deb, exe or dmg

Well, I use jill... Much better than juliaup on Linux...

**Comment by AbhimanyuAryan** [comment]
> > I think a lot of people use `juliaup` instead of installing directly using deb, exe or dmg
> 
> Well, I use jill... Much better than juliaup on Linux...

Not sure link please?

**Comment by ufechner7** [comment]
https://github.com/johnnychen94/jill.py

**Comment by AbhimanyuAryan** [comment]
> https://github.com/johnnychen94/jill.py

But this has Python as deps. Windows doesn't really come with Python pre-installed so isn’t it even easier to get juliaup with a winget and not have Python at all?

**Comment by ufechner7** [comment]
Yes, on Windows juliaup might be easier... But you get the latest release only with some delay because you need to update juliaup first which is not required with jill...
And on Linux I have installed Python by default anyways... And jill works much better in China...

**Comment by ufechner7** [comment]
And Sys.BINDIR() would not work on Linux due to the use of symbolic links...
